sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,191 @@
+DROP TABLE IF EXISTS holiday_dates;
+CREATE TABLE holiday_dates (
+  id INT UNSIGNED NOT NULL,
+  date_id TINYINT UNSIGNED NOT NULL,
+  date_value INT UNSIGNED NOT NULL,
+  PRIMARY KEY (id, date_id)
+);
+
```

Comment:
i'd suggest to not save packed time in database, use unix timestamp instead and transform them when loading in core

Reply:
I decided to keep the format because this is plain dbc data extracted directly from the 7.2 client (except for the Darkmoon stuff because the Faire has been reworked)